assets/queries/openAPI/server_url_uses_undefined_variables/query.rego

```diff
@@ -71,6 +71,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	server := doc.components.responses[r].links[l].server
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)